pep-0669.rst

```diff
@@ -1,12 +1,12 @@
 PEP: 669
 Title: Low Impact Monitoring for CPython
 Author: Mark Shannon <mark@hotpy.org>
-Status: Draft
+Status: Accepted
 Type: Standards Track
 Content-Type: text/x-rst
 Created: 18-Aug-2021
 Post-History: 07-Dec-2021
```

Comment:
```suggestion
Post-History: `07-Dec-2021 <https://mail.python.org/archives/list/python-dev@python.org/thread/VNSD4TSAM2BM64FJNIQPAOPNEGNX4MDX/>`__,
              `10-Jan-2022 <https://discuss.python.org/t/pep-669-low-impact-monitoring-for-cpython/13018>`__,
```

Add/update Post-History

Reply:
(I saw you also posted it https://mail.python.org/archives/list/python-dev@python.org/thread/XWLJ7JXODYRKAPKQCXO7IQ2AME6JM7HQ/#XWLJ7JXODYRKAPKQCXO7IQ2AME6JM7HQ , but the thread didn't get any replied whereas the Discourse thread saw plenty of activity before and after that, so I didn't add that one but you can if you want, of course).